scout/adapter/mongo/query.py

```diff
@@ -474,63 +474,88 @@ def coordinate_filter(self, query, mongo_query):
 
         return mongo_query
 
-    def sv_coordinate_query(self, query):
-        """Adds genomic coordinated-related filters to the query object
-            This method is called to buid coordinate query for sv variants
-
-        Args:
-            query(dict): a dictionary of query filters specified by the users
-            mongo_query(dict): the query that is going to be submitted to the database
+    def get_position_query(self, chrom: str, start: int, end: int) -> dict:
+        """Helper function that returns a dictionary containing start and stop coordinates.
```

Comment:
Let's move (or copy if you prefer) the descriptions of the other cases (chrom != end_chrom) up into the docstring as well!

Reply:
Copy. Good idea- I like these comments to be where they are since they clarify a rather complex situation in that query